Warehouse.Persistence.Abstractions/Constants/DapperGenericConstants.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Warehouse.Persistence.Abstractions.Constants;
+
+public class DapperGenericConstants
+{
+    public const string GetById = """SELECT * FROM "{0}" WHERE "Id" = @Id""";
```

Comment:
`public static string GetById(string entity ) => """SELECT * FROM "{entity}" WHERE "Id" = @Id""";`
may be something like that

Reply:
I prefer to use current solution to be consistent with all other queries. I will just arrange the query to different rows.